### PR TITLE
added the header and footer component

### PR DESCRIPTION
**Pull request by c-blaq**

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import Link from "next/link";
+import Header from "./styled/Header.styled";
+import styled from "styled-components";
+
+const Nav = styled.nav`
+  padding: 0.75rem 0 1.6rem 1rem;
+  display: flex;
+  flex-direction: column;
+  gap: 0;
+  width: 30%;
```
Comment (funabab) [inline src/components/Header.js]: This should be in the styled folder

File: public/images/socials/facebook-icon.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg width="11" height="18" viewBox="0 0 11 18" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (funabab) [inline public/images/socials/facebook-icon.svg]: Will prefer you use react icons instead, it's already included as a dependency.

File: pages/tag/[tag].jsx [hunk 3]
```diff
@@ -3,80 +3,87 @@ import {
...
   getArticleSlugFromPath,
-} from '~utils/article'
-import { promises as fs } from 'fs'
-import matter from 'gray-matter'
-import Link from 'next/link'
+} from "~utils/article";
+import { promises as fs } from "fs";
+import matter from "gray-matter";
+import Link from "next/link";
+import HeaderContent from "~components/Header";
```
Comment (funabab) [inline pages/tag/[tag].jsx:10]: This should be imported with the name Header rather than HeaderContent

File: src/components/Header.js (hunk 1 above)
Comment (c-blaq) [reply src/components/Header.js]: Okay, will fix it

File: public/images/socials/facebook-icon.svg (hunk 2 above)
Comment (c-blaq) [reply public/images/socials/facebook-icon.svg]: Noted, thanks

**Pull request merged**